src/components/ResourceList/ResourceList.scss

```diff
@@ -119,13 +119,65 @@ $item-wrapper-loading-height: rem(64px);
 }
 
 // Force long sort options to be truncated
-.SortWrapper,
-.SortWrapper > * {
-  @include breakpoint-before(resource-list(breakpoint-small)) {
+// .SortWrapper,
+// .SortWrapper > * {
+// overflow: hidden;
+// box-sizing: border-box;
+// outline-offset: 10px;
+// outline: 3px solid color('indigo');
+// border: border-width() solid color('indigo');
+// &:focus {
+//   box-shadow: inset 0 0 0 2px color('red');
+// }
+// }
+
+// $stacking-order: (
+//   item: 10,
+//   focused: 20,
+// );
+
+// .SortWrapper,
+// .SortWrapper > * {
+//   overflow: hidden;
+// }
+
+.SortWrapper {
```

Comment:
@beefchimi this is a possible solution... tested on chrome, firefox & safari 

but would love some input 🥬

Reply:
Okay, so this can actually be fixed rather easily 😄 

You can replace ALL of your changed code in this file with:

```scss
.SortWrapper {
  @include layout-flex-fix;

  // Required to apply styles to the <Labelled /> component
  > * {
    max-width: 100%;
  }
}
```

That is the most simple solution. Two things to potentially alter:

**1. Apply "layout fix" to all children**
Personally, I would apply that `layout-flex-fix` mixin to ALL `children` of `.HeaderContentWrapper`. So, you could instead do:

```scss
.HeaderTitleWrapper,
.CheckableButtonWrapper,
.AlternateToolWrapper,
.SortWrapper,
.SelectButtonWrapper {
  @include layout-flex-fix;
}

.SortWrapper {
  > * {
    max-width: 100%;
  }
}
```

**2. Try to avoid reaching into other components to override styles**
The `> *` selector is specifically targeting the `.Labelled` direct child. It would be preferable if we did not have to apply this snowflake override style... so I'm wondering if `max-width: 100%` can live comfortably on the `.Labelled` selector within `Labelled.scss`. I think its worth experimenting with that change to see if it causes issues any where else `<Labelled />` is used.

**Explanation**
We need to utilize this `layout-flex-fix` mixin on direct children ("flex items") of a `display: flex` parent in order to get truncation to work correctly. This is a very common problem/solution! All that mixin is doing is applying the following styles:

```scss
min-width: 0;
max-width: 100%;
```

This seems... silly, because those prop + values seem obvious and should be default behaviour.... right?!?! The short answer is: **flex box items will not shrink below their minimum content size**. So, if the minimum content size is computed as the full width of a sentence, it cannot be _below that width_ in order to truncate. In comes out handy "flex fix" styling to say _"well actually min-width can be `0`"_.

For a better explanation, our very own @dfmcphee wrote a blog post on it!
https://dfmcphee.com/flex-items-and-min-width-0/

Reply:
thanks so much @beefchimi! really appreciate that. 

fyi for posterity's sake: 

adding the `max-width: 100%` on the `.Labelled` selector in `Labelled.scss` results in:
![image](https://user-images.githubusercontent.com/38010031/80998699-5e8b5800-8e11-11ea-94b3-aa391d4aef70.png)

adding the `@include layout-flex-fix;` to all children of `.HeaderContentWrapper` results in: 
![image](https://user-images.githubusercontent.com/38010031/80998824-9d211280-8e11-11ea-9122-c9e5a71c930c.png)

so i'm going with your very fist suggestion. thanks again for your input!

Reply:
We _could_ improve this further with some tweaks to the other `HeaderContentWrapper`, mainly to that `SelectWrapper` so that it has a `min-width` of _at least_ the:
- `width` of a `checkbox` +
- `spacing()`

But just so other reviews know - the label of that "select checkbox" will truncate as well. Its just that Caroline's example is _soooo long_ that it goes beyond the text label and starts encroaching onto the `checkbox` input. Realistically, we probably wouldn't end up in the scenario.

@carolopolo would you mind opening an issue with your 2nd screenshot to track this? You can link to this comment + include any relevant context.